File: 6/6.c
#include <stdio.h>
#include <math.h>


int main(void) {
  
  long sum_squares = 0;
  for (int x=1; x<101; x++) {
    sum_squares += powl(x, 2); 
  }
  long squared_sums = 0;
  for (int x=1; x<101; x++) {
    squared_sums += x;
  }
  squared_sums = powl(squared_sums, 2);

  printf("Problem 6 Answer: %ld\n", squared_sums - sum_squares);
}

File: 6/6.go
package main

import "fmt"

func main() {
	sum_squares := 1
	for i := 2; i < 101; i++ {
		sum_squares += i * i
	}

	square_sums := 0
	for i := 1; i < 101; i++ {
		square_sums += i
	}
	square_sums *= square_sums

	fmt.Println("Problem 6 Answer:", square_sums-sum_squares)
}
